admin/doc-requirements.txt

```diff
@@ -1,3 +1,3 @@
 Sphinx == 1.6.3
--e git+https://github.com/ceph/sphinx-ditaa.git#egg=sphinx-ditaa
+-e git+https://github.com/ceph/sphinx-ditaa.git@py3#egg=sphinx-ditaa
```

Comment:
if I read this correctly, this means the Python3 branch? is that the one where the warning has been removed?

Reply:
> if I read this correctly, this means the Python3 branch?

yes, see also the commit message

> the py3 branch is created to track the upstream's master branch

yes.

> is that the one where the warning has been removed?

yes. and since the PRs (https://github.com/baloo/sphinx-ditaa/pull/9 and https://github.com/baloo/sphinx-ditaa/pull/8)  for upstreams have been merged.  i sync'ed it with upstream's master. so it's now identical to https://github.com/baloo/sphinx-ditaa 's master now.